talk/owt/sdk/conference/conferenceclient.cc

```diff
@@ -300,8 +300,8 @@ void ConferenceClient::Join(
                                   "Failed to parse current user's info"));
                 on_failure(std::move(e));
               });
+              break;
```

Comment:
My understanding the whole "for" loop above can be removed.

Reply:
Agree with u, I will remove the for-loop.